### PR TITLE
Fix theme validator

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
A change in the theme validator here: https://github.com/mattermost/desktop/pull/3833 caused an issue where the theme required all fields. Commonly `sidebarTeamBarBg` and `codeTheme` are missing, so this PR allows the fields to be blank (which allows for a default to load).

```release-note
NONE
```

## Change Impact: 🟠 Medium

**Regression Risk:** The change relaxes theme validation in a shared validator module (src/common/Validator.ts), altering how multiple theme fields accept empty strings. While this fixes valid themes previously rejected, it modifies a widely-used validation contract without accompanying unit tests, increasing the chance of unintended acceptance of malformed or incomplete theme objects in downstream consumers.

**QA Recommendation:** Moderate manual QA and unit test additions recommended. Add targeted unit tests for the theme schema (including empty-string, omitted, and valid non-empty values) and perform manual checks that theme loading, defaults, and UI rendering behave correctly across common platforms and edge cases.

*Generated by CodeRabbitAI*

### CODE REVIEW

**Comment by devinbinnie** [comment]
@amyblais This was a regression in a change for the v6.2.1 dot release

**Comment by amyblais** [comment]
> This was a regression in a change for the v6.2.1 dot release

Thanks for the heads-up!

File: src/common/Validator.ts [hunk 1]
```diff
@@ -369,30 +369,30 @@ export function ipcValidate<E, A extends unknown[], R>(
...
+    sidebarTextActiveColor: Joi.string().allow(''),
+    sidebarHeaderBg: Joi.string().allow(''),
+    sidebarTeamBarBg: Joi.string().allow(''),
+    sidebarHeaderTextColor: Joi.string().allow(''),
+    onlineIndicator: Joi.string().allow(''),
+    awayIndicator: Joi.string().allow(''),
+    dndIndicator: Joi.string().allow(''),
+    mentionBg: Joi.string().allow(''),
+    mentionColor: Joi.string().allow(''),
     centerChannelBg: Joi.string(),
```
[the remark this answers is not in the document]
Comment (devinbinnie) [reply src/common/Validator.ts:386]: This is intentional - we need this to be not blank for certain other functions to work.

**Review by edgarbellot — APPROVED** [review]
